src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -106,31 +106,53 @@ static String describeType(ParameterType type, String prefix) throws Exception {
         }
         return typeInfo.toString();
     }
-
+    
     private static String describeErrorType(ParameterType type) {
         return StringEscapeUtils.escapeHtml(type.getActualType().toString());
     }
 
     private static String generateAttrHelp(DescribableParameter param) throws Exception {
         StringBuilder attrHelp = new StringBuilder();
         String help = param.getHelp();
-        if (help != null && !help.equals("")) {
-            attrHelp.append(helpify(help)).append("\n");
-        }
         try {
+            if (help != null && !help.equals("")) {
+                attrHelp.append(helpify(help)).append("\n");
+            }
             String typeDesc;
+            ParameterType type = param.getType();
+            if (!(type instanceof AtomicType)) {
+                typeDesc = describeType(param.getType(), "");
+                attrHelp.append("<ul>").append(typeDesc).append("</ul>");
+            }
+        } catch (RuntimeException | Error ex) {
+            LOG.log(Level.WARNING, "Restricted description of attribute "
```

Comment:
Capturing all errors always feels like a code smell to me. Which ones are ignorable and which ones are actually bad? This will mean that'll "randomly" have empty help and you'll have to go look at the logs to see why.

Its easier to maintain when you handle very specific errors so you know what kind of bad data it can expect

Reply:
Also, if you have a definite failure, maybe there's some parsing problem with the data that we can figure out here.  I think this is a copy of what we have done for this sort of help before.  So I'm sort of fine with skipping for no.  But if we could find an example of a failure, that would be really interesting to look at to try to troubleshoot together.